### PR TITLE
Add Probot as a curated topic

### PR DESCRIPTION
**Pull request by tcbyrd**
- [X] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [X] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [X] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [X] My folder contains a `*.png` image (if applicable) and `index.md`
- [X] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

Probot is an extension of GitHub Apps, created and maintained by both Hubbers and the larger GitHub community. As more Apps are created, we want to ensure users can easily find Probot Apps and related extensions using topics.

cc @bkeepers in case you'd like to add/edit anything. I used March 2017 for the release based on the first [release](https://github.com/probot/probot/releases) (0.2.0).

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/probot/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+created_by: Brandon Keepers, Bex Warner
+display_name: Probot
+github_url: https://github.com/probot/
+logo: probot.png
+released: March 2017
+short_description: GitHub Apps to automate and improve your workflow
```
Comment (cheshire137) [inline topics/probot/index.md]: Could you add a period?

File: topics/probot/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+created_by: Brandon Keepers, Bex Warner
```
Comment (cheshire137) [inline topics/probot/index.md]: You could add a `related` list including github-apps and bot, if you wanted.

**Comment by tcbyrd** [comment]
@cheshire137 Thanks for the review! I wasn't sure how much the machines would glean automatically! 🤖 LMK if you have any other tips to make this awesome. 💪

File: topics/probot/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+related: github-apps, bot, node-github
+display_name: Probot
+github_url: https://github.com/probot/
+logo: probot.png
+released: March 2017
+short_description: GitHub Apps to automate and improve your workflow.
```
Comment (cheshire137) [inline topics/probot/index.md]: How about changing the short_description so it mentions Probot itself, not just the end result from using Probot? Maybe "Probot lets you create GitHub Apps to automate and improve your workflow."

Comment (tcbyrd) [reply topics/probot/index.md]: Done!

**Pull request merged**